Receptai.client/middleware/auth.ts

```diff
@@ -1,28 +1,25 @@
 import axios from "axios";
-import type { UserCookie } from "~/typescript/types";
+import type { UserCookie } from "@/typescript/types";
 
 export default defineNuxtRouteMiddleware(async () => {
   const config = useRuntimeConfig();
 
   const TastyBytes_user = useCookie<UserCookie | null>("TastyBytes_user");
 
   if (TastyBytes_user.value) {
-    const userObj: UserCookie = TastyBytes_user.value;
-
     try {
-      axios
-        .get(`${config.public.baseURL}/api/v1/user/me`, {
-          headers: { Authorization: `Bearer ${userObj.token}` },
-        })
-        .then(() => {
-          return;
-        })
-        .catch(async () => {
-          return navigateTo("/user/login");
-        });
-    } catch (e) {
-      console.log("Auth", e);
-      TastyBytes_user.value = null;
+      const response = await axios.get(
+        `${config.public.baseURL}/api/v1/user/me`,
+        {
+          headers: { Authorization: `Bearer ${TastyBytes_user.value.token}` },
+        }
+      );
+      if (!response.data.roles.includes("ROLE_ADMIN")) {
```

Comment:
Čia logika skaitant nebus kad prašys log in jei vartotojo rolė ne ADMIN? Reiktų paaiškint kada auth.ts yra iškviečiama